lms/resources/_js_config/__init__.py

```diff
@@ -208,6 +208,7 @@ def enable_file_picker_mode(self, form_action, form_fields):
         self._config.update(
             {
                 "mode": JSConfig.Mode.FILE_PICKER,
+                "product": self._get_product_info(),
```

Comment:
It would be nice if we explained what this is for? I can't tell looking at this what it does or why we need it.